### ISSUE TITLE
Validate nested arrays throws exception

### ISSUE THREAD

**Issue by ZehuaZhang** [YStrauch/swagger-object-validator#18]
Great package, using flawlessly except one problem.
Cannot validate pure array schema, had to wrap around in an object for now

Schema
```
{
  type: "array",
  "x-nullable": false,
  items: {
    type: "array",
    items: {
      type: "string",
      minLength: 1,
    },
  },
  description: "nested arrays"
}
```
Validation 
```
validator.validateModel(
      [
        ['1'],['2']
      ],
     schema,
```
Error
```
TypeError: Cannot set property 'arrayPos' of undefined
    at /Users/node_modules/swagger-object-validator/lib/validators/ArrayValidator.js:40:48
    at Array.forEach (<anonymous>)
    at validateArray (/Users/node_modules/swagger-object-validator/lib/validators/ArrayValidator.js:38:10)
    at /Users//node_modules/swagger-object-validator/lib/validators/ModelValidator.js:58:29
From previous event:
    at Object.validateModel (/Users/node_modules/swagger-object-validator/lib/validators/ModelValidator.js:26:10)
    at /Users/node_modules/swagger-object-validator/lib/handler.js:46:67
From previous event:
    at /Users/node_modules/swagger-object-validator/lib/handler.js:46:18
```

**Comment by ZehuaZhang** [comment]
Thank you for this!

**Comment by YStrauch** [comment]
Oh I forgot to comment - this is fixed in version 1.3.0
